tests/entry_page_test.js

```diff
@@ -27,88 +27,18 @@ Scenario('Verify confirmations when both experiments matching', async ({ I }) =>
-        }else{
-            I.assertNotContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }
-    });
-});
-
-Scenario('Verify no confirmations when first experiment (non entry page) matching', async ({ I }) => {
-   //Act
-   I.amOnPage('/clone.html');
-   I.wait(2);
-   let activeKeys = await I.executeScript(async () => {
-       let keys = await evolv.client.getActiveKeys();
-       let filteredKeys = keys.current.filter(key => key.split('.').length < 3);
-       return filteredKeys.map(key => key.split('.')[1]);
-   })
-   let experimentsConfig = await utils.getConfiguration();
-   let confirmations = await I.executeScript(async () => {
-       let confirmationsArray = [];
-      
-       if(evolv.context.remoteContext.experiments.confirmations){
-
-           evolv.context.remoteContext.experiments.confirmations.forEach(element => {
-               confirmationsArray.push(element.cid);
-           })
-           return confirmationsArray;
-       }
-       else {
-           return [];
-       };
-   });
-   const entryPoints = await utils.getEntryPoints(activeKeys, experimentsConfig);
-
-   //Assert
-   I.assertEqual(confirmations.length, 0);
-   entryPoints.forEach(element => {
-        I.assertEqual(element.isEntryPoint, false);
-   });
-});
-
-Scenario('Verify confirmations when second experiment (entry page) matching', async ({ I }) => {
-    //Act
-    I.amOnPage('/clone1.html');
-    I.wait(2);
-    let confirmations = await I.executeScript(async () => {
-        let confirmationsArray = [];
-       
-        if(evolv.context.remoteContext.experiments.confirmations){
-
-            evolv.context.remoteContext.experiments.confirmations.forEach(element => {
-                confirmationsArray.push(element.cid);
-            })
-            return confirmationsArray;
-        }
-        else {
-            return [];
-        };
-    });
-
-    let activeKeys = await I.executeScript(async () => {
-        let keys = await evolv.client.getActiveKeys();
-        let filteredKeys = keys.current.filter(key => key.split('.').length < 3);
-        return filteredKeys.map(key => key.split('.')[1]);
-    })
+    const allActiveKeysEntryPointInfo = await utils.getEntryPoints(activeKeys, experimentsConfig);
 
-    let experimentsConfig = await utils.getConfiguration();
- 
-    const entryPoints = await utils.getEntryPoints(activeKeys, experimentsConfig);
-   
     //Assert
-    entryPoints.forEach(element => {
-        if(element.isEntryPoint){
-            I.assertContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }else{
-            I.assertNotContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }
+    I.assertTrue(confirmations.length > 0);
+    allActiveKeysEntryPointInfo.forEach(element => {
+        confirmations.forEach(confirmation => {
+            if(element.isEntryPoint) {
+                I.assertTrue(confirmation.indexOf(element.id) > -1);
```

Comment:
Should be able to use 'includes'
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

This is not supported in IE -- so we'd want to verify that these scenario pieces of code run outside of IE itself (allI mean by that is do a test run in browserstack)

Reply:
`includes` won't work. We want to match a partial entry. Includes only works if it's the entire value.

Reply:
`indexOf` is supported by IE11. I can remove the arrow function syntax if that's the issue.